tests/framework/extensions/clusters/clusters.go

```diff
@@ -1,11 +1,31 @@
 package clusters
 
 import (
+	"fmt"
+	"strings"
+
 	apisV1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
 	rkev1 "github.com/rancher/rancher/pkg/apis/rke.cattle.io/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
 )
 
+var ClusterReadyStatusFunc = func(event watch.Event) (ready bool, err error) {
```

Comment:
there is no need to assign this to a var, you could do a normal function declaration here.

Reply:
Fixed